src/custom/pages/Rewards/index.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import Page, { Title, Content, GdocsListStyle } from 'components/Page'
+import styled from 'styled-components'
+import { Trans } from '@lingui/macro'
+import CowsImg from 'assets/images/cows.png'
+
+const ButtonGroup = styled.div`
+  display: flex;
+  align-items: center;
+  margin-top: 1rem;
+`
+
+const FlexContainer = styled.div`
+  display: flex;
+  align-items: flex-start;
+`
+
+const SecondaryButton = styled.div`
+  width: 350px;
+  height: 65px;
+  border: 1px solid #3d3d3d;
+  box-sizing: border-box;
+  border-radius: 9px;
+  font-size: 18px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 0 0.5rem;
+  cursor: pointer;
+`
+
+const PrimaryButton = styled(SecondaryButton)`
```

Comment:
I'd say to use the `ButtonPrimary` and `ButtonSecondary` already in `/custom/components/Button`.

@alfetopito wdty?

Reply:
Yep, agreed.
Let's try to reuse as much as possible of existing components/styles.
No need to match the styles in the wireframes at this point.